docs/quickstart/docker.md

```diff
@@ -192,8 +201,9 @@ Instead of build Zeppelin distribution package and docker image everytime during
 Zeppelin can run locally (such as inside your IDE in debug mode) and able to run Interpreter using [DockerInterpreterLauncher](https://github.com/apache/zeppelin/blob/master/zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterLauncher.java) by configuring following environment variables.
 
 
+1. zeppelin-site.xml
```

Comment:
zeppelin-env.sh ?

Reply:
1. `ZEPPELIN_RUN_MODE ` already exist in https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L617
2. `ZEPPELIN_DOCKER_CONTAINER_IMAGE ` already exist in https://github.com/apache/zeppelin/blob/master/conf/zeppelin-site.xml.template#L647

i modify `Environment variable` to `Configuration variable`.